upload-wheel.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+wheel="$1"
+url="s3://everlaw-build-artifacts/fasttext/$(basename "$wheel")"
+
+if aws s3 ls "$url" &>/dev/null; then
+    >&2 echo "$url already exists!"
+    exit 0
+fi
+
+aws s3 cp --only-show-errors "$wheel" "$url"
```

Comment:
The runner has permission to do this because of https://github.com/Everlaw/servers/pull/39903.